webrender/src/renderer.rs

```diff
@@ -1481,6 +1517,93 @@ impl Renderer {
+        while let Ok(msg) = self._debug_server.debug_rx.try_recv() {
+            match msg {
+                DebugMsg::FetchBatches(sender) => {
+                    let mut batch_list = BatchList::new();
+
+                    if let Some(ref frame) = self.current_frame {
+                        if let Some(ref frame) = frame.frame {
+                            for pass in &frame.passes {
+                                for target in &pass.alpha_targets.targets {
+                                    batch_list.push("[Clip] Clear", target.clip_batcher.border_clears.len());
+                                    batch_list.push("[Clip] Borders", target.clip_batcher.borders.len());
+                                    batch_list.push("[Clip] Rectangles", target.clip_batcher.rectangles.len());
+                                    for (_, items) in target.clip_batcher.images.iter() {
+                                        batch_list.push("[Clip] Image mask", items.len());
+                                    }
+                                }
+
+                                for target in &pass.color_targets.targets {
+                                    batch_list.push("[Cache] Vertical Blur", target.vertical_blurs.len());
+                                    batch_list.push("[Cache] Horizontal Blur", target.horizontal_blurs.len());
+                                    batch_list.push("[Cache] Box Shadow", target.box_shadow_cache_prims.len());
+                                    batch_list.push("[Cache] Text Shadow", target.text_run_cache_prims.len());
+                                    batch_list.push("[Cache] Lines", target.line_cache_prims.len());
+
+                                    for batch in target.alpha_batcher
+                                                       .batch_list
+                                                       .opaque_batches
+                                                       .iter()
+                                                       .rev() {
+                                        batch_list.push(batch.key.kind.debug_name(), batch.instances.len());
+                                    }
+
+                                    for batch in &target.alpha_batcher
+                                                        .batch_list
+                                                        .alpha_batches {
+                                        batch_list.push(batch.key.kind.debug_name(), batch.instances.len());
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    let json = serde_json::to_string(&batch_list).unwrap();
+                    sender.send(json).ok();
+                }
+            }
+        }
+    }
+
+    fn handle_debug_command(&mut self, command: DebugCommand) {
+        match command {
+            DebugCommand::EnableProfiler(enable) => {
```

Comment:
Do you mean remove the keyboard shortcuts? I think it makes sense to keep those for quick debugging - or did you mean something else?